blaze-core/src/main/scala/org/http4s/blazecore/util/ChunkWriter.scala

```diff
@@ -41,11 +40,13 @@ private[util] object ChunkWriter {
         ByteBuffer.wrap(rr.result.getBytes(ISO_8859_1))
       } else ChunkEndBuffer
     }
-    unsafeRunAsync(f) {
-      case Right(buffer) =>
-        IO { promise.completeWith(pipe.channelWrite(buffer).map(Function.const(false))); () }
-      case Left(t) =>
-        IO { promise.failure(t); () }
+    F.start {
```

Comment:
This never runs the action in start, LOL. 

Do you mind reverting this? This isn't a necessary change and I'd rather not touch blaze core here.